src/pages/ReviewForm/ReviewForm.js

```diff
@@ -57,6 +60,7 @@ export default function FormComponent() {
               <div>
                 <Rating //user can review the quality of his dealer by awarding the number of stars (out of 5)
                 setReviews={[currentValue, setCurrentValue]} //pass the params down to child class (Rating) under component
+                ratinghover={ratinghover}
```

Comment:
can just change `ratinghover` here directly to prevent declaring another constant to get `ratingHover` under Rating.js